packages/console/src/pages/GetStarted/components/Skeleton/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import * as styles from './index.module.scss';
+
+const Skeleton = () => (
+  <>
+    {Array.from({ length: 5 }).map((_, stepIndex) => (
```

Comment:
Instead of having this will it be better to use:
```ts
[...Array.from().keys()].map(key => </>)
```

Reply:
updated